### PR TITLE
chore: narrow scope of .gitignore for proposals/a:upgrade-next folder

### LINKED ISSUE #9846: Narrow Scope of .gitignore for proposals/a:upgrade-next folder
## What is the Problem Being Solved?
The current `a3p-integration/.gitignore` settings for the `proposals/a:upgrade-next` directory are too broad. We need to revise this to target only the generated files specifically.

## Description of the Design
- Add a targeted `.gitignore` file for the `proposals/a:upgrade-next` folder. This file should be configured to ignore only the generated files. The following subdirectories should be added in the `.gitignore`:

- `proposals/a:upgrade-next/add-LEMONS/`
- `proposals/a:upgrade-next/add-OLIVES/`
- `proposals/a:upgrade-next/upgrade-bank/`
- `proposals/a:upgrade-next/upgrade-provisionPool/`

### PR DESCRIPTION
**Pull request by rabi-siddique**
closes #9846

### CODE REVIEW

File: a3p-integration/.gitignore
Comment (mhofman) [inline a3p-integration/.gitignore]: Please move those in a new git ignore file in proposals/a:upgrade-next as it will facilitate maintenance.

Comment (rabi-siddique) [reply a3p-integration/.gitignore]: Good point. Updated

File: a3p-integration/proposals/a:upgrade-next/.gitignore
Comment (mhofman) [inline a3p-integration/proposals/a:upgrade-next/.gitignore:1]: I believe the paths need to be relative to the location of the ignore file

**Review by mhofman — APPROVED** [review]
Approved assuming the path will be fixed, which I expect will be needed to satisfy CI. Use `force:integration` if you want to test before selecting a merge strategy.

File: a3p-integration/proposals/a:upgrade-next/.gitignore
Comment (rabi-siddique) [reply a3p-integration/proposals/a:upgrade-next/.gitignore:1]: Thanks for pointing out this silly mistake.

Comment (rabi-siddique) [reply a3p-integration/proposals/a:upgrade-next/.gitignore:1]: I'll merge after addressing the CI failures related to the endo-integration-test:
https://github.com/Agoric/agoric-sdk/actions/runs/10261407685

Comment (mhofman) [reply a3p-integration/proposals/a:upgrade-next/.gitignore:1]: We should fix this asap. Currently it prevents seamless modifications to the `proposals/a:upgrade-next` folder (without `git add -f`) and some work is scheduled that requires updating that folder.

**Pull request merged**
